src/coffee/delivery-export.coffee

```diff
@@ -0,0 +1,147 @@
+{SphereClient} = require 'sphere-node-sdk'
+Promise = require 'bluebird'
+_ = require 'underscore'
+Csv = require 'csv'
+objectPath = require 'object-path'
+
+class DeliveryExport
+  constructor: (@options, @logger) ->
+    @client = new SphereClient @options.client
+    @stats = @_getDefaultStats()
+
+    @options.export = _.defaults @options.export or {},
+      where: ''
+      perPage: 500
+
+  _getDefaultStats: ->
+    _.clone
+      loadedOrders: 0
+      exportedDeliveries: 0
+      exportedRows: 0
+
+  _getCsvHeader: ->
+    [
+      'orderNumber',
+      'delivery.id',
+      '_itemGroupId',
+      'item.id',
+      'item.quantity',
+      'parcel.id',
+      'parcel.length',
+      'parcel.height',
+      'parcel.width',
+      'parcel.weight',
+      'parcel.trackingId',
+      'parcel.carrier',
+      'parcel.provider',
+      'parcel.providerTransaction',
+      'parcel.isReturn'
+    ].join(',')
+
+  _castRowsToCsv: (rows) ->
+    new Promise (resolve, reject) ->
+      Csv().from rows
+        .on 'error', reject
+        .to.string resolve
+
+  # take delivery objects and map them to CSV rows
+  _mapDeliveriesToRows: (orderNumber, deliveries) ->
```

Comment:
I am not sure this method _name_ matches what it does

From what I am seeing, this just increases the counter of _exportedDeliveries_